db/data/20210910152831_update_funding_type_on_funding_methods.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class UpdateFundingTypeOnFundingMethods < ActiveRecord::Migration[6.1]
+  def up
+    SEED_BURSARIES.each do |b|
```

Comment:
This migration is called `UpdateFundingTypeOnFundingMethods ` - I expected it do something like, rather than re-seeding:

```
def up
 FundingMethod.update_all(funding_type: :bursary)
end
```